relayer/pkg/starknet/utils.go

```diff
@@ -28,6 +28,10 @@ func PadBytes(a []byte, length int) []byte {
 	return a
 }
 
+func FeltToBytes(felt *caigotypes.Felt) []byte {
+	return PadBytes(felt.Bytes(), FeltLength)
+}
```

Comment:
Should we just fix this in caigo? And make `felt.Bytes()` return the right format